docs/management/connectors/action-types/email.asciidoc

```diff
@@ -229,3 +246,59 @@ secrets:
 <2> Many organizations support use of your email address as your username.
     Check with your system administrator if you receive
     authentication-related failures.
+
+Before you create an email connector for Microsoft Exchange, you must create and register the client integration application on the https://go.microsoft.com/fwlink/?linkid=2083908[Azure portal]:
+
+[role="screenshot"]
+image::management/connectors/images/exchange-register-app.png[Register client application for MS Exchange]
+
+On the Azure portal, open *Manage > API permissions*, and then define the permissions for the registered application to send emails. Refer to the https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=http#permissions[documentation] for the Microsoft Graph API.
+You can configure it on the Azure portal application page, Manage - API permissions:
+[role="screenshot"]
+image::management/connectors/images/exchange-api-permissions.png[MS Exchange API permissions]
+
+Add the "Mail.Send" permission for Microsoft Graph.  The permission appears in the list with the status "Not granted for <your Azure active directory>":
+[role="screenshot"]
+image::management/connectors/images/exchange-not-granted.png[MS Exchange "Mail.Send" not granted]
+
+Click *Grant admin consent for <your Azure active directory>*.
+[role="screenshot"]
+image::management/connectors/images/exchange-grant-confirm.png[MS Exchange grant confirmation]
+
+Confirm the grant. The status for the "Mail.Send" permission changes to granted.
```

Comment:
should this be:

Confirm that the status for the "Mail.Send" permission is now granted